src/arm/linux/cpuinfo.c

```diff
@@ -802,7 +802,7 @@ static bool parse_line(
 			} else if (memcmp(line_start, "Revision", key_length) == 0) {
 				size_t value_length = value_end - value_start;
 				if (value_length > CPUINFO_REVISION_VALUE_MAX) {
-					cpuinfo_log_info(
+					cpuinfo_log_debug(
```

Comment:
Hmm, this should be warning, isn't it? And if limit have become larger, should we just increase it instead?

Reply:
Upgraded to warning, the limit being fixed seems to be an artifact of the struct holding a fixed length array. We could change it to dynamically allocate? In any case, happy to do that in an isolated PR so that this one is focused on fixing the log levels.

Reply:
So I can't quite tell what the hardware revision or name is used for, because it's just an arbitrary string - surely, no one should be reading this value to make a decision? They should be requesting the micro-arch enums if they needed this information and needed to switch behavior on it.

Assuming that's true, then I'd suggest this should be debug - it's unfortunate that it cuts the name off, but shouldn't affect proper usage of the library / isn't something a user needs to actively fix.

Reply:
Do you mind splitting this PR into two: let's downgrade others (as indeed empty lines or lines with no separators feels benign) and keep discussion on those in a separate issue PR.
 IMO if identifier needs to be truncated, it warrants a warning, but I've never seen it in practice

Reply:
Ack, reverting these back to 'info' and creating separate PR.